### PR TITLE
chore: upgrade Staging Kubernetes to 1.22

### PR DESCRIPTION
**Pull request by patheard**
# Summary
Upgrade Staging Kubernetes cluster to `v1.22` and
bump the node group AMI release version.

# Related
* cds-snc/notification-planning#555

### CODE REVIEW

**Review by iokpala — APPROVED** [review]
LGTM

**Comment by patheard** [comment]
Blocked by cds-snc/notification-planning#643

File: env/staging/eks/terragrunt.hcl [hunk 1]
```diff
@@ -68,11 +68,11 @@ inputs = {
   firehose_waf_logs_iam_role_arn         = dependency.common.outputs.firehose_waf_logs_iam_role_arn
   cloudfront_assets_arn                  = dependency.cloudfront.outputs.cloudfront_assets_arn
   eks_cluster_name                       = "notification-canada-ca-staging-eks-cluster"
-  eks_cluster_version                    = "1.21"
+  eks_cluster_version                    = "1.22"
   eks_addon_coredns_version              = "v1.8.4-eksbuild.1"
   eks_addon_kube_proxy_version           = "v1.21.2-eksbuild.2"
   eks_addon_vpc_cni_version              = "v1.11.0-eksbuild.1"
-  eks_node_ami_version                   = "1.21.5-20220420"
+  eks_node_ami_version                   = "1.22.6-20220421"
```
[the remark this answers is not in the document]
Comment (patheard) [reply env/staging/eks/terragrunt.hcl:75]: No good reason at all - I'll get that updated

**Review by sastels — APPROVED** [review]
I guess changing the k8s versions causes things to change for those certs and iam resources? 🤷

**Comment by patheard** [comment]
> I guess changing the k8s versions causes things to change for those certs and iam resources? 🤷

Yup, that's it - I've seen the exact same `terraform plan` for the other cluster version upgrades.

**Pull request merged**